### PR TITLE
Fixing TODO visible in docs

### PR DESCRIPTION
**Pull request by moxiegirl**
ping @docker/compose-maintainers 

Reported by a user. 

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

File: docs/networking.md [hunk 1]
```diff
@@ -79,9 +79,11 @@ You can specify which one to use with the `--x-network-driver` flag:
 
     $ docker-compose --x-networking --x-network-driver=overlay up
 
+<!--[metadata]>
 ## Multi-host networking
 
 (TODO: talk about Swarm and the overlay driver)
+<![end-metadata]-->
```
Comment (thaJeztah) [inline docs/networking.md]: Aren't these stripped before the docs is generated?

Comment (moxiegirl) [inline docs/networking.md]: nope only the metadata

**Comment by dnephin** [comment]
LGTM

**Comment by shin-** [comment]
LGTM!

**Pull request merged**